samples/rest/BingWebSearchV7.py

```diff
@@ -7,33 +7,41 @@
 import os
 from pprint import pprint
 import requests
+from dotenv import load_dotenv
+
+# Load the environment variables from .env file
+load_dotenv()
 
 '''
 This sample makes a call to the Bing Web Search API with a query and returns relevant web search.
 Documentation: https://docs.microsoft.com/en-us/bing/search-apis/bing-web-search/overview
 '''
+AUTH_HEADER=os.environ.get('BING_SEARCH_V7_AUTHORIZATION_HEADER', 'Ocp-Apim-Subscription-Key')
```

Comment:
This is fine, but the subscription key header name hasn't changed since the Cognitive Services days and I don't foresee future changes there, so I think hard-coding the name is probably better.

Reply:
Done: aaee62ec4862618af122513ffc88f05ec2669a79